server/views/pages/vary-approve/request-changes.njk

```diff
@@ -40,7 +43,7 @@
                         classes: 'govuk-visually-hidden'
                     },
                     errorMessage: validationErrors | findError('reasonForReferral'),
-                    value: licence.reasonForReferral | fillFormResponse(formResponses.reasonForReferral)
+                    value: formResponses.reasonForReferral
                 }) }}
```

Comment:
No longer pre-populating a reason - each reason is a new one - unless the result of a validation error.